### DISCUSSION TITLE
choice of outbound per user

### DISCUSSION THREAD

**Discussion by boss1819** [apernet/hysteria#723] [Q&A] [answered]
I would like to use a different outbound per client (user/pass). so far I do not think ACL rules can achieve this.
is there any other way to do it?
can I run multiple hysteria services on same server, listening to different ports (to be able to assign a different outbound each)?
thanks

**Comment by tobyxdd** [comment]
ACL doesn't support per-user rules. But yes, you can run multiple instances as long as they listen on different ports.

**Reply by boss1819** [reply, marked as answer]
thanks
